libraries/src/main/java/com/baeldung/flink/WordCount.java

```diff
@@ -0,0 +1,21 @@
+package com.baeldung.flink;
+
+import org.apache.flink.api.java.DataSet;
+import org.apache.flink.api.java.ExecutionEnvironment;
+import org.apache.flink.api.java.aggregation.Aggregations;
+import org.apache.flink.api.java.tuple.Tuple2;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class WordCount {
+
+    public static DataSet<Tuple2<String, Integer>> startWordCount(ExecutionEnvironment env, List<String> lines) throws Exception {
+        DataSet<String> text = env.fromCollection(lines);
+
+        return text.flatMap(new LineSplitter())
+                .groupBy(0)
+                .aggregate(Aggregations.SUM, 1);
```

Comment:
yes, i know but aggregate is more flexible because you can pass other types of operations, so i prefer to show that method, ok?